scipy/_lib/tests/test__util.py

```diff
@@ -647,65 +641,3 @@ def test_rng_deterministic(self, method, arg_name):
         res3 = method(self, **{arg_name: None})
         assert_equal(res2, res1)
         assert_equal(res3, res1)
-
-
-class TestLazywhere:
```

Comment:
Does `apply_where` still pass this test

Reply:
I've added them to `array_api_extra`. all green.

Reply:
Ah, great!